network/iptables.md

```diff
@@ -15,6 +15,61 @@ manipulate this table manually. If you need to add rules which load before
 Docker's rules, add them to the `DOCKER-USER` chain. These rules are loaded
 before any rules Docker creates automatically.
 
+### Add a DOCKER-USER filter chain to allow persistent rules 
+This can be useful if you need to pre-populate `iptables` rules that need to be in place before Docker runs. The following example creates a new chain named `FILTERS` in which network traffic from `INPUT` AND `DOCKER-USER` is put.
```

Comment:
could you wrap this to ~80 chars, and add whitespace after the heading and before the code block?

```suggestion

This can be useful if you need to pre-populate `iptables` rules that need to be
in place before Docker runs. The following example creates a new chain named
`FILTERS` in which network traffic from `INPUT` AND `DOCKER-USER` is put.

```

Reply:
Yes - done!